templates/staff/application/researcher_edit.html

```diff
@@ -0,0 +1,107 @@
+        {{ error }}
+      {% /alert %}
+    {% endfor %}
+  {% endif %}
+
+  {% #card title="Researcher contact information" class="max-w-3xl" container=True %}
+    {% #form_fieldset class="grid gap-y-6 !min-w-0 w-full max-w-prose" %}
+      {% form_legend text="Researcher contact information" class="sr-only" %}
+
+      {% form_input field=form.name label="Researcher name" name="name" %}
+      {% form_input field=form.job_title label="Researcher job title" name="job_title" %}
+      {% form_input field=form.email label="Researcher work email address" name="email" type="email" %}
+      {% form_input field=form.telephone label="Mobile phone number" name="telephone" %}
+      {% form_input field=form.github_username label="GitHub username" name="github_username" %}
+      {% form_input field=form.daa label="DAA" name="daa" %}
+    {% /form_fieldset %}
+  {% /card %}
+
+  {% #card title="Secure results server" class="max-w-3xl" container=True %}
+    {% #form_fieldset class="grid gap-y-2 !min-w-0 w-full max-w-prose" %}
+      {% form_legend text="Does this researcher require access to the OpenSAFELY secure results server?" %}
+
+      {% for value, label in form.does_researcher_need_server_access.field.choices %}
+        {% with id=forloop.counter0|stringformat:"s" %}
+          {% if form.does_researcher_need_server_access.value == value %}
+          {% var checked=True %}
+          {% else %}
+          {% var checked=False %}
+          {% endif %}
+          {% form_radio name="does_researcher_need_server_access" id="id_does_researcher_need_server_access"|add:label|add:"_"|add:id|slugify label=label value=value checked=checked %}
+        {% endwith %}
+      {% endfor %}
+    {% /form_fieldset %}
+  {% /card %}
+
+  {% #card title="Mobile phone" class="max-w-3xl" container=True %}
+    <p class="prose mb-4">
+      This is required to set up Virtual Private Network (VPN) access to access the results server.
+    </p>
+
+    {% #form_fieldset class="grid gap-y-2 !min-w-0 w-full max-w-prose" %}
+      {% form_legend text="What type of smartphone does this researcher have?" %}
+
+      {% for value, label in form.phone_type.field.choices %}
+        {% with id=forloop.counter0|stringformat:"s" %}
+          {% if form.phone_type.value == value %}
+          {% var checked=True %}
+          {% else %}
+          {% var checked=False %}
+          {% endif %}
+          {% form_radio name="phone_type" id="id_"|add:label|add:"_"|add:id|slugify label=label value=value checked=checked %}
+        {% endwith %}
+      {% endfor %}
+    {% /form_fieldset %}
+  {% /card %}
+
+  {% #card title="Safe researcher training" class="max-w-3xl" container=True container_class="grid gap-y-6 max-w-prose" %}
+    {% #form_fieldset class="grid gap-y-2 !min-w-0 w-full max-w-prose" %}
+      {% form_legend text="Has this researcher undertaken safe researcher training?" %}
+
+      {% for value, label in form.has_taken_safe_researcher_training.field.choices %}
```

Comment:
If there's a validation error somewhere on the page when I press submit, then the page refreshes and loses this value.

I can then submit without answering this question - I'm not sure if that's intentional or not?